Nest/00. Plan.md

```diff
@@ -0,0 +1,67 @@
+# 1 Introduction
+## 1.1 Package manager
+
+## 1.2 Packages and dependencies
+
+## 1.3 Repositories
+
+## 1.4 Package's syntax
+
+# 2 Nest
+## 2.1 Nest's options
+
+## 2.2 Nest commands
+### 2.2.1 Synchronizing the local cache
+### 2.2.2 Troubleshooting
+
+## 2.3 Installing and uninstalling a package
+### 2.3.1 Installing a package
+### 2.3.2 Uninstalling a package
+### 2.3.3 Troubleshooting
+
+## 2.4 Updating your system
+### 2.4.1 Troubleshooting
+
+## 2.5 Conclusion
+
+# 3 Some more advanced concepts
+## 3.1 More about the repositories
+### 3.1.1 The official repositories
+### 3.1.2 The unofficial repositories
+### 3.1.3 Changing the repositories' configuration file
+
+## 3.2 The cache
+
+## 3.3 Version requirement
+
+# 4 Finest
+## 4.1 Key concepts
+### 4.1.1 The cache
+### 4.1.2 The transactions
+### 4.1.3 The dependency graph
+### 4.1.4 Groups
+### 4.1.5 Example
+
+## 4.2 Finest's options
+
+## 4.3 Finest's commands
+### 4.3.1 Installing and removing packages through requirements
```

Comment:
This title is quite long, can we shorter it? If not, that's fine.

Reply:
`Managing packages through requirements`?